tests/validation/rules/requiredIf.spec.ts

```diff
@@ -48,7 +48,7 @@ describe('validation/rules/requiredIf', () => {
     })
 
     const { validation } = useV(data, {
-      v: { requiredIf: requiredIf(() => condition.value) }
+      v: { required: requiredIf(() => condition.value) }
```

Comment:
Why? 👀

Reply:
No strong reason.
I just follow how we do in our application codebase.
I can revert if you prefer.

Reply:
Ah OK, but that's not true though. Try searching `requiredIf` and you will see lots of our app codes uses `requiredIf: ` syntax 😅 

I'm fine with this 👍  Just wanted know if you thought this key means anything and changed intentionally to fix something or not 👀  The key can be anything so I don't really care either 😳